### ISSUE TITLE
Cannot update to recent version

### ISSUE THREAD

**Issue by Jurek-Raben** [el1t/uBlock-Safari#51]
If I download and then install (double click) the recent build in my macos sierra 10.12.4, I click on "trust", but then still version "1.11.3.1" is listed under Safari plugins.

How do I update?

**Comment by ruchernchong** [comment]
To update to the 1.12 version, you have to compile it using the source code from here yourself.

**Comment by Jurek-Raben** [comment]
Ok, but the download itself shows 1.12.5b?
 https://github.com/el1t/uBlock-Safari/releases/download/1.12.5b/uBlock0.safariextz

**Comment by ruchernchong** [comment]
@JurekRaben Oh, then just download that. I checked back a few days ago, I did not see that version as binaries in the release section.

Do be warned that there are many problems with v1.12 as per the number of issues in this repo right now.

**Comment by Jurek-Raben** [comment]
But that download doesn't work as update, that's what this topic is about :)

**Comment by MysticalOS** [comment]
The gallery version is probably not signed same as the untrusted distro. It likely won't "upgrade" but install as a separate extension if dev hasn't signed it exactly the same.
Basically just use the stand alone ones or only use the gallery one.

**Comment by el1t** [comment]
@MysticalOS is correct in that the gallery one was uploaded by another person who signed the extension themselves—not from the releases page. The releases are all signed with a personal developer certificate. An update to the gallery has been submitted, but Apple hasn't updated yet.

**Comment by vassudanagunta** [comment]
Yes, the Gallery version is an unauthorized and unverified binary put up by @hartator and signed by *his* certificate **without permission!**  I created Issue #69 after waiting 6 months for @hartator to do the right thing... He hasn't.

**Comment by vassudanagunta** [comment]
To help out the maintainer, @el1t, can we close this issue? The problem has been solved (#86) has been answered, and we are tracking the unauthorized extension and website in the referenced issue above. Ellis is working on submitting an update to the App Store (which has replaced the Extension Gallery).
